internal/docstore/dynamodocstore/dynamo_test.go

```diff
@@ -14,12 +14,20 @@
 
 package dynamodocstore
 
+// To create the tables and indexes needed for these tests, run create_tables.sh in
```

Comment:
nit: let's move this part to right above `const` declarations.

Reply:
Done.